desktop/core/src/desktop/js/catalog/optimizer/ApiStrategy.ts

```diff
@@ -0,0 +1,188 @@
+// Licensed to Cloudera, Inc. under one
```

Comment:
At some point, renaming 'optimizer' --> 'assistant'?

Reply:
Indeed! Should be easy to adjust.